lib/cfnvpn/templates/lambdas/auto_route_populator/quotas.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+
+ROUTE_TABLE_QUOTA_CODE = 'L-401D78F7'
+AUTH_RULE_TABLE_QUOTA_CODE = 'L-9A1BC94B'
+EC2_SERVICE_CODE = 'ec2'
+IN_PROGRESS = ['PENDING', 'CASE_OPENED']
+
+def get_route_count(endpoint) -> int:
+  client = boto3.client('ec2')
+  response = client.describe_client_vpn_routes(
```

Comment:
Is it worth paginating this?

Reply:
seeing as the default limit is 10 routes i dont think we need to worry about that yet